### ISSUE TITLE
Remove exit option from Confirmed on Trezor pill modal

### ISSUE THREAD

**Issue by MiroslavProchazka** [trezor/trezor-suite#11802]
Follow up on [#11771](https://github.com/trezor/trezor-suite/issues/11771), please remove the cross on the pill modal upon the SOL send form, after user confirms the transaction on trezor and the modal from https://github.com/trezor/trezor-suite/pull/11515 is waiting on SOL Tx confirmation. 
This needs to be done because user might click on the cross and break the TX confirmation. 

![image](https://github.com/trezor/trezor-suite/assets/152851523/5b44cf0a-f559-415f-9b82-b8816e192119)

**Comment by Hannsek** [comment]
It shouldn't be removed. The pill tells users that they confirmed that on Trezor, not that the Transaction is confirmed.
The whole logic is not ideal, but this is how it is done now and creating exceptions won't help our users.

**Comment by Hannsek** [comment]
I don't think that is the ideal solution. If we remove the cross, user won't be able to exit the flow if he wants to do it.
Plus this would be _another_ exception in our codebase which is 🫤.
